src/controllers/v2/auth.controller.js

```
@@ -0,0 +1,167 @@
+          phone: z
+            .string({ required_error: "Phone number is required." })
+            .length(10, "Phone number must be exactly 10 digits long."),
+          availability: z.boolean({
+            required_error: "Availability is required.",
+          }),
+          vehicleType: z.enum(["BIKE", "CYCLE"], {
+            required_error:
+              "Vehicle type is required. Must be one of BIKE, CYCLE.",
+          }),
+        })
+        .parse(req.body);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)